akka-stream-tests/src/test/scala/akka/stream/scaladsl/AttributesSpec.scala

```diff
@@ -168,6 +178,19 @@ class AttributesSpec extends StreamSpec(ConfigFactory.parseString(
 
   }
 
+  "attributes on a composed graph" must {
+    "allow for overriding the attributes inside the composed graph" in {
+      val attributes =
+        Source.fromGraph(new AttributesSource().addAttributes(whateverAttribute("on-stage")))
+          .toMat(Sink.head)(Keep.left)
+          .addAttributes(whateverAttribute("on-composed").important)
```

Comment:
this is how it is intended to be used. I think this makes very much sense. It's a way of saying "I know what I'm doing, I override what was inside that other part"

Reply:
Yeah, I think it may be enough for most cases, without the seals complexity/flexibility. I'd bikeshed the name a little bit though I think.

Reply:
I thought about `force` at some point but I think that implies that the outermost wins (which ofc could also be an option to discuss)

Reply:
Yeah force implies outermost wins, so does important as well hmm, does lock sound interesting?

Reply:
Otherwise i guess important would be fine